en/sim_gazebo_gz/index.md

```diff
@@ -174,6 +174,46 @@ This can be ignored:
 
 :::
 
+### Change Simulation Speed
+
+PX4 SITL can be run faster or slower than real-time when using Gazebo.
+
+The speed factor is set using the environment variable `PX4_SIM_SPEED_FACTOR`.
+For example, to run the Gazebo simulation of the X500 frame at 2 times the real time speed:
+
+```sh
+PX4_SIM_SPEED_FACTOR=2 make px4_sitl gz_x500
+```
+
+To run at half real-time:
+
+```sh
+PX4_SIM_SPEED_FACTOR=0.5 make px4_sitl gz_x500
+```
+
+You can apply the factor to all SITL runs in the current session using `EXPORT`:
+
+```sh
+export PX4_SIM_SPEED_FACTOR=2
+make px4_sitl gz_x500
+```
+
+::: tip
+To avoid PX4 detecting data link timeouts, increase the value of param [COM_DL_LOSS_T](../advanced_config/parameter_reference.md#COM_DL_LOSS_T) proportional to the simulation rate.
+For example, if `COM_DL_LOSS_T` is 10 in realtime, at 10x simulation rate increase to 100.
```

Comment:
I don't think this applies anymore, since px4 time is in lockstep with simulation time